docker/windows/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM microsoft/windowsservercore:1709 AS base
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV PYTHON_VERSION 3.6.5
+ENV NODEJS_VERSION 8.11.1
+ENV DOTNETCORESDK_VERSION 2.1.4
```

Comment:
can you also upgrade the versions in the linux Dockerfile to match this Windows Dockerfile?

Reply:
I think we should put all version in an external file for easier maintenance